src/main/java/org/springframework/data/jpa/provider/PersistenceProvider.java

```diff
@@ -355,6 +368,69 @@ public String getCountQueryPlaceholder() {
 	}
 
 	/**
+	 * Creates a dynamic {@link EntityGraph} from the given {@link JpaEntityGraph} information. 
```

Comment:
What's the reason these methods live in `PersistenceProvider`? There's nothing provider specific in here. Maybe a left over from the state of using provider specific API?